src/mixer/samplerbank.h

```diff
@@ -19,7 +19,7 @@ class SamplerBank : public QObject {
             PlayerManager* pPlayerManager);
 
     bool saveSamplerBankToPath(const QString& samplerBankPath);
-    bool loadSamplerBankFromPath(const QString& samplerBankPath);
+    bool loadSamplerBankFromPath(const QString& samplerBankPath, bool initializing = false);
```

Comment:
How about: 
```suggestion
    bool loadSamplerBankFromPath(const QString& samplerBankPath, bool SkipCreateingHiddenSamplers = false);
```
To document what the function does not when it should be called.